source/refined-github.css

```diff
@@ -2,7 +2,7 @@
 	/* 2020-12-15: these variables only exist for GHE #3798 */
 	--github-green: var(--color-text-success, #28a745);
 	--github-red: var(--color-text-danger, #cb2431);
-	--github-border-color: var(--color-border-primary, #e1e4e8);
+	--github-border-color: var(--color-border-muted, #e1e4e8);
```

Comment:
This is also affecting other features. Here's the before/after

<img width="155" alt="Screen Shot 9" src="https://user-images.githubusercontent.com/1402241/135693105-bc1cfb6b-3bdd-4412-ae03-4659063cff43.png"> <img width="168" alt="Screen Shot 7" src="https://user-images.githubusercontent.com/1402241/135693108-0ea18bd8-590f-4cef-aef3-3aea695b87ba.png">


<img width="274" alt="Screen Shot 8" src="https://user-images.githubusercontent.com/1402241/135693114-61bff2ba-7cfc-4ad4-9b94-91823b021346.png"> <img width="270" alt="Screen Shot 4" src="https://user-images.githubusercontent.com/1402241/135693117-77973d07-8576-49d6-9ca3-7bac9b9bdcfc.png">

Reply:
Like https://github.com/sindresorhus/refined-github/pull/3806